main/views.py

```diff
@@ -102,11 +118,33 @@ def reply(request, food_id, review_id):
 
 @login_required
 def cart(request):
-    status = get_object_or_404(Status, name='cart')
-    bill = Bill.objects.prefetch_related('item_set').filter(status=status).first()
+    bill, cart_items, _ = get_cart(request)
     
     context = {
         "cart": bill,
-        "items": bill.item_set.all
+        "items": cart_items
     }
     return render(request, 'cart/cart.html', context)
+
+@login_required
+def add_to_cart(request):
+    bill, cart_items, _ = get_cart(request)
+    food = get_object_or_404(Food, id=request.POST.get('id'))
+    action = ''
+
+    if cart_items.filter(food=food):
```

Comment:
nên dùng `exists()` để check
```suggestion
    if cart_items.filter(food=food).exists():
```